transifex/requirements.txt

```diff
@@ -2,4 +2,4 @@ Django==1.11.2
 edx-i18n-tools==0.3.9
 PyGithub==1.29
 PyYAML==3.12
```

Comment:
Do we still need pyYAML?

Reply:
i will check if we are not using it in any other place i will remove it. we are no longer using it for openedx.yaml.